src/api/routes/v2/schemas.ts

```diff
@@ -0,0 +1,70 @@
+import { Type, Static } from '@sinclair/typebox';
+import { TypeCompiler } from '@sinclair/typebox/compiler';
+import { ResourceType, pagingQueryLimits } from '../../../api/pagination';
+
+// ==========================
+// Parameters
+// ==========================
+
+const OffsetParam = Type.Integer({
+  minimum: 0,
+  title: 'Offset',
+  description: 'Result offset',
+});
+
+export const BlockLimitParam = Type.Integer({
+  minimum: 1,
+  maximum: pagingQueryLimits[ResourceType.Block].maxLimit,
+  default: pagingQueryLimits[ResourceType.Block].defaultLimit,
+  title: 'Block limit',
+  description: 'Blocks per page',
+});
+
+const BurnBlockHashParam = Type.RegExp(/^[0]{8}[a-fA-F0-9]{56}$/, {
```

Comment:
Should this also allow for a `0x` prefix? It's not common for bitcoin block hashes, but we use it for the rest of the hash strings, i.e. `/^(0x)?[a-fA-F0-9]{64}$/i`

Reply:
Done